bkcore.coffee/controllers/GamepadController.coffee

```diff
@@ -23,8 +23,8 @@ class GamepadController
   updateAvailable: ->
     return false if not @active
     gamepads = if navigator.getGamepads then navigator.getGamepads() else navigator.webkitGetGamepads()
-    return false if not gamepads?[0]
-    gp = gamepads[0]
+    return false if not gamepads
+    gp = (pad for pad in gamepads when pad?.mapping is 'standard')[0]
```

Comment:
What if the gamepads list only has non-standard pads? We probably need a check before selecting `[0]` or change the line below to `gp?.buttons?`